encrypted-media-respec.html

```diff
@@ -1661,30 +1663,28 @@
                         <dd>
                           <p>Run the following steps:</p>
                           <ol>
-                            <li><p>Clear all stored session data associated with this object, including the <a def-id="sessionId"></a> and license destruction record.</p>
-                              <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
-                            </li>
                             <li>
                               <p>
-                                Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.
+                                Use <var>cdm</var> to clear all stored session data associated with this object, including the <a def-id="sessionId"></a>
+                                and license destruction record.
                               </p>
+                              <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
                             </li>
-                            <li><p>Run the <a def-id="session-close-algorithm"></a> algorithm on this object.</p></li>
+                            <li><p>Run the <a def-id="close-session-algorithm"></a> algorithm on this object.</p></li>
```

Comment:
I implemented the above suggestion, removing Close Session (since it was only used by `close()`) and adding a normative requirement to the Key Session definition that the licenses and keys are deleted when the session is closed.